vcd/resource_vcd_external_network_test.go

```diff
@@ -169,3 +169,85 @@ resource "vcd_external_network" "{{.ExternalNetworkName}}" {
   retain_net_info_across_deployments = "false"
 }
 `
+
+func TestAccVcdExternalNetworkResourceNotFound(t *testing.T) {
+	preTestChecks(t)
+
+	// This test invokes go-vcloud-director SDK directly
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	if !usingSysAdmin() {
+		t.Skip("TestAccVcdExternalNetworkBasic requires system admin privileges")
```

Comment:
It's referencing another test, so we could either put `TestAccVcdExternalNetworkResourceNotFound` or maybe better:
```suggestion
		t.Skip(t.Name() + " requires system admin privileges")
```